pkgs/tools/networking/linux-router/default.nix

```
@@ -0,0 +1,102 @@
+# SOURCE: https://github.com/NixOS/nixpkgs/pull/92787/files
+
+{ stdenv, lib, pkgs, fetchFromGitHub
```

```suggestion
{ stdenv, lib, fetchFromGitHub
```

---

I guess we then have to add  `makeWrapper` here

```suggestion
{ stdenv, lib, fetchFromGitHub, makeWrapper
```